tsan_suppressions.txt

```diff
@@ -2,5 +2,33 @@
 race:cppmicroservices::ServiceListeners::RemoveServiceListener
 race:cppmicroservices::ServiceTracker*
 
+# BundleRegistry races and double lock false-positives
```

Comment:
a link to the github issue, https://github.com/google/sanitizers/issues/1259, would be helpful here.